requirements.txt

```diff
@@ -2,4 +2,7 @@ click
 progress
 pyyaml
 simplejson
-cryptography
+# Pin to cryptography==3.4 since it is the last dependency version that
+# requires to be build with Rust and was breaking the snap.
+# More info on issue: https://github.com/canonical/hotsos/issues/326
```

Comment:
```suggestion
# Pin to cryptography==3.4.8 since the cryptography-3.4.x branch is the
# last branch that can be built without the Rust toolchain..
# More info on issue: https://github.com/canonical/hotsos/issues/326
# [TODO] Bump cryptography to more recent release
```

Reply:
Fixed on https://github.com/canonical/hotsos/commit/985654fca5885b803c178de214a597569b76882b